team/index.html

```diff
@@ -588,7 +588,6 @@ <h1>Developers</h1>
 						<span>PDX Git Engineer</span>
 					</div>
 				</div>
-
```

Comment:
Why?

Reply:
because first I wanted to add another team member but i changed my choice

Reply:
Revert unrelated changes